.travis.yml

```diff
@@ -9,5 +9,3 @@ script:
   - pub run dart_dev format --check
   - pub run dart_dev analyze
   - pub run dart_dev test
-  - pub run dart_dev coverage --no-html
-  - bash <(curl -s https://codecov.io/bash) -f coverage/coverage.lcov
```

Comment:
coverage would hang forever in travis for some reason, so I finally just had to leave it off